.github/workflows/triage.yml

```diff
@@ -29,4 +29,24 @@ jobs:
                   github-token: ${{secrets.GITHUB_TOKEN}}
                   script: |
                       const issue = { owner: context.issue.owner, repo: context.issue.repo, issue_number: context.issue.number }
-                      github.issues.addLabels({...issue, labels: ['area: triage', 'theme: undefined']})
+                      github.issues.listLabelsOnIssue({...issue}).then(response => {
+                        const labels = response.data
+                        let missingLabel = []
```

Comment:
Very minor, but I think we could write this concisely as `let missingLabel = [], missngArea = true, missingTheme = true`

Reply:
thanks, done :)